src/tools/genpolicy/rules.rego

```diff
@@ -1092,12 +1092,23 @@ match_caps(p_caps, i_caps) {
 }
 
 ######################################################################
+
+check_directory_traversal(i_path) {
+    not regex.match("(^|/)..($|/)", i_path)
```

Comment:
@burgerdev , are there other advantages to using this regex, except for a shorter policy text?

If there aren't other advantages, this other version is easier for me to understand:

check_directory_traversal(i_path) {
    contains(i_path, "../") == false
    endswith(i_path, "/..") == false
    i_path != ".."
}

Do you feel strongly that we should use the regex?

Reply:
I wrote the regex when I realized the above is wrong: consider `i_path == /foo/bar/etc../file`. Let me add that to the test cases.